scripts/build-docs.js

```diff
@@ -61,6 +63,12 @@ components.forEach(component => {
     content += `:::\n`;
   }
 
+  if (component.cation) {
```

Comment:
You have a typo here (should be `caution`)

Reply:
🤦‍♂️ fixed